src/ex2/item_manager.js

```diff
@@ -0,0 +1,53 @@
+import PokemonClient from "./PokemonClient.js";
+
+class ItemManager {
+	constructor() {
+		this.id = 0;
+		this.itemArray = [];
+		this.pokemonClient = new PokemonClient();
+	}
+	addListItem(item) {
+		const { id, value } = this.createId(item);
+		this.itemArray = [...this.itemArray, { id, value }];
+		return this.itemArray;
+	}
+	removeItem(itemId) {
+		this.itemArray = [
+			...this.itemArray.filter((item) => item.id !== parseInt(itemId)),
+		];
+
+		return this.itemArray;
+	}
+
+	async pokemonIdsHendeling(pokeID) {
```

Comment:
This function name isn't very clear, first of all it gets a single id in you have "ids" in the name.
Renaming it to something like "fetchPokemonData" can make it a lot clearer.